### PR TITLE
Re-organize main nav on desktop

### PR DESCRIPTION
**Pull request by anandaroop**
https://artsyproduct.atlassian.net/browse/BUY-141
https://artsyproduct.atlassian.net/browse/BUY-143
https://artsyproduct.atlassian.net/browse/BUY-151
https://artsyproduct.atlassian.net/browse/BUY-154

nav final

- Re-orders top nav menu items visually
- Re-prioritizes top nav menu items "snapping" into **More** menu
- Removes most items from **More**
- Adds Blog to footer links 
- Updates TAGP footer link to go to /categories

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: src/desktop/components/main_layout/footer/upper.jade [hunk 1]
```diff
@@ -12,6 +12,7 @@
   .mlf-upper-column
     h4 About Artsy
     a( href='/about' ) About
+    a( href='https://medium.com/artsy-blog' ) Blog
     a( href='/about/jobs' ) Jobs
     a( href='http://artsy.github.com/open-source/', target='_blank' ) Open Source
```
Comment (oxaudo) [inline src/desktop/components/main_layout/footer/upper.jade]: Unrelated: Surprised this one is not `https://`

Comment (anandaroop) [reply src/desktop/components/main_layout/footer/upper.jade]: Oh yeah good point. Maybe this link predates https by default on Medium. Anyway I'll change it 👍

Comment (anandaroop) [reply src/desktop/components/main_layout/footer/upper.jade]: Whoops deleted earlier comment where I misread the link you were talking about — will fix this one in any case.

**Comment by anandaroop** [comment]
Updated with @briansw 's feedback and ready for final review/merge @oxaudo

**Pull request merged**